docker-compose.dev.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  app:
+    build: .
+    tty: true
+    depends_on:
+      - mysql
+    ports:
+      - "3000:3000"
+  mysql:
+    image: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+    volumes:
+      - mysql-vol
```

Comment:
`mysql-vol` would be better `mysql-data`. In addition, it's a good idea to mount it to `/var/lib/mysql`.

Reply:
Please refer to `docker-compose.yml` of main branch.

Reply:
Thx. Changed as mentioned.